core/src/main/java/io/questdb/griffin/SqlCodeGenerator.java

```diff
@@ -1762,7 +1762,7 @@ private RecordCursorFactory generateFill(QueryModel model, RecordCursorFactory g
             for (int i = 0, n = model.getBottomUpColumns().size(); i < n; i++) {
                 final QueryColumn col = model.getColumns().getQuick(i);
                 final ExpressionNode ast = col.getAst();
-                if (Chars.equals(ast.token, "timestamp_floor")) {
+                if (isTimestampFloorKeyword(ast.token)) {
```

Comment:
nit: a `Chars#equalsIgnoreCase()` call would be enough here. A 15 chars-long function matcher isn't very readable.